jsx/Form.js

```diff
@@ -477,15 +477,15 @@ class SelectElement extends Component {
     // and retain formatting. If label prop is not provided at all, the input
     // element will take up the whole row.
     let label = null;
-    let inputClass = 'col-sm-12';
+    let inputClass = this.props.noMargins ? '' : 'col-sm-12';
     if (this.props.label && this.props.label != '') {
       label = (
         <label className="col-sm-3 control-label" htmlFor={this.props.label}>
           {this.props.label}
           {requiredHTML}
         </label>
       );
-      inputClass = 'col-sm-9';
+      inputClass = this.props.noMargins ? '' : 'col-sm-9';
```

Comment:
i would suggest leaving this line unchanged. i think the select element looks better with input classes col-sm-3 and col-sm-9 if it has a label

<img width="482" alt="Screen Shot 2020-06-12 at 2 12 14 PM" src="https://user-images.githubusercontent.com/17415878/84534034-3831c580-acb7-11ea-967b-bf83c9eae811.png">

Reply:
the last two rows is with line 488 untouched. what do you think @maltheism ?

Reply:
@zaliqarosli yah seems good. I'll change it back.